staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -185,6 +205,25 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	switch requestInfo.Verb {
+	case "create", "update", "patch":
+		found := false
+		for _, v := range crd.Status.StoredVersions {
+			if crdInfo.storageVersion == v {
+				found = true
+				break
+			}
+		}
+		if !found {
+			crd.Status.StoredVersions = append(crd.Status.StoredVersions, crdInfo.storageVersion)
+			crd, err = r.crdGetter.CustomResourceDefinitions().UpdateStatus(crd)
+			if err != nil {
```

Comment:
In any case this would need a Conflict retry logic.

Reply:
done